tests/test_user_management.py

```diff
@@ -1,19 +1,25 @@
 import json
 import unittest
+import pytest
 
 import requests
 
-from pylana.api import get_user_information
+from os import path
 
+from pylana.api import get_user_information
+from pylana import create_api
 
-class TestGetUserInformation(unittest.TestCase):
+@pytest.mark.skipif(not path.exists('./tests/admin_config.json'), 
+                    reason="admin privileges necessary")
+class TestUsersAPI(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls) -> None:
-        with open('./tests/config.json') as f:
+        with open('./tests/admin_config.json') as f:
             cls.credentials = json.load(f)
+        cls.api = create_api(verify=True, **cls.credentials)
 
-    def test_success(self):
+    def test_user_by_token_success(self):
```

Comment:
This test should be in another file `test_api.py`, since it is defined in `api.py` and used there.